src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/mapper/LimitedPartnershipMapper.java

```diff
@@ -16,5 +17,15 @@ public interface LimitedPartnershipMapper {
 
   LimitedPartnershipSubmissionDao dtoToDao(LimitedPartnershipSubmissionDto dto);
 
-  DataDao map(DataDto dataDto);
+  DataDao mapData(DataDto dataDto);
+
+
+  default String mapPartnershipNameEndingToString(PartnershipNameEnding nameEnding){
```

Comment:
You might find these aren't needed once the `JsonCreator` annotation is added. We didn't have default mapper methods in ROE.

Reply:
The default mapper methods map the value rather than the name of the enum. Otherwise we would be persisting the name. The JsonCreator may be needed when the Json is deserialised in the controller but I think that's outside the scope of this PR.